src/lrulfu.c

```diff
@@ -18,7 +18,8 @@
 // Current time in seconds (24 least significant bits).  Designed to roll over.
 static uint32_t LRUGetClockTime(void) {
 #if LRU_CLOCK_RESOLUTION == 1000
-    return (uint32_t)(server.unixtime & LRULFU_MASK);
+    time_t unixtime = atomic_load_explicit(&server.unixtime, memory_order_relaxed);
+    return (uint32_t)(unixtime & LRULFU_MASK);
```

Comment:
Is `time_t` larger than 32 bits? If yes, then it makes sense to make it atomic on 32-bit.

Need to declare the `unixtime` field in the valkeyServer struct as `_Atomic` too.